src/apps/relay/mainrelay.c

```diff
@@ -95,6 +95,7 @@ turn_params_t turn_params = {
     "",                     /*ca_cert_file*/
     "turn_server_cert.pem", /*cert_file*/
     "turn_server_pkey.pem", /*pkey_file*/
+    0,                      /*rpk_enabled*/
```

Comment:
sorry to nitpick, but this would be `false` from `<stdbool.h>`

Reply:
Ok fixed, and just fyi, I don't have write access, so git is not letting me merge the pull request.